x-pack/qa/password-protected-keystore/src/test/java/org/elasticsearch/password_protected_keystore/ReloadSecureSettingsWithPasswordProtectedKeystoreRestIT.java

```diff
@@ -78,10 +78,16 @@ public void testReloadSecureSettingsWithEmptyPassword() throws Exception {
             assertThat(entry.getValue(), instanceOf(Map.class));
             final Map<String, Object> node = (Map<String, Object>) entry.getValue();
             assertThat(node.get("reload_exception"), instanceOf(Map.class));
-            assertThat(ObjectPath.eval("reload_exception.reason", node), anyOf(
-                equalTo("Provided keystore password was incorrect"),
-                equalTo("Keystore has been corrupted or tampered with")));
-            assertThat(ObjectPath.eval("reload_exception.type", node), equalTo("security_exception"));
+            if (inFipsJvm()) {
```

Comment:
The "keystore is tampered with is also thrown in fips, see https://github.com/elastic/elasticsearch/pull/57050

Reply:
For this specific case, i.e. empty password, the error message should always be 
`Error generating an encryption key from the provided password` because it is generated without even reading the keystore file as opposed to other error messages. With this being said, I like @tvernum's suggestion as well. So we are settled here.